public/assets/js/main.js

```diff
@@ -20,18 +22,53 @@
         }
     })();
 
-    (function selectInputContent() {
+    (function selectTextCopyToClipboard() {
         var selector = 'input[type="text"]';
         var el = document.querySelectorAll(selector);
+        var origHelpBlockText = 'Click to copy';
 
+        // Loop through all `input`s
         for (var i = 0, len = el.length; i < len; i++) {
-            el[i].addEventListener('focus', function() {
+            el[i].addEventListener('focus', function(e) {
+                e.preventDefault();
                 this.select();
-            });
-            el[i].addEventListener('mouseup', function (event) {
-                event.preventDefault();
-            });
+
+                var clipboardSnippets = new Clipboard(this, {
+                    target: function (trigger) {
+                        return trigger;
+                    }
+                });
+
+                clipboardSnippets.on('success', function (e) {
+                    var helpBlock = {};
+
+                    if (e.trigger.parentElement.nextElementSibling &&
```

Comment:
Perhaps set `var nextSibling = e.trigger.parentElement.nextElementSibling;` to avoid repeating the verbose reference.